.github/workflows/pytorch-version-tests.yml

```diff
@@ -26,6 +26,10 @@ jobs:
             python-version: 3.9 
           - pytorch-version: 1.6.0
             python-version: 3.9 
+          - pytorch-version: 1.7.1
```

Comment:
Let's add a comment why we excluding these confs

Reply:
ok,  
for 1.7.1 and 1..8.1 it is temporary right?

Reply:
> Let's add a comment why we excluding these confs

#### Like:

-  `python-version:` is list of versions of python.
-  `pytorch-version: ` is list of versions of PyTorch.
- and in `exclude:` we can give the particular PyTorch and python version to remove from testing.
Does sound right?

Reply:
No, a comment explaning why we excluding 3.9/1.7.1 and 3.9/1.8.1 and put a link on https://github.com/pytorch/ignite/issues/2383

Reply:
okay.